Cargo.toml

```diff
@@ -26,7 +26,7 @@ base64 = "0.10"
 bincode = "1.0"
 bitcoin = { version = "0.21", features = ["use-serde"] }
 bitcoin_hashes = "0.7.1"
-configure_me = "0.3.3"
+configure_me = "0.4.0"
```

Comment:
Was getting unsatisfied deps issues related to this and `configure_me_codegen` so I just updated them both to their latest.